allennlp/modules/augmented_lstm.py

```diff
@@ -67,6 +70,19 @@ def __init__(self,
         else:
             self.input_linearity = torch.nn.Linear(input_size, 4 * hidden_size, bias=True)
             self.state_linearity = torch.nn.Linear(hidden_size, 4 * hidden_size, bias=True)
+        self.reset_parameters()
+
+    def reset_parameters(self):
+        # Use sensible default initializations for parameters.
+        block_orthogonal(self.input_linearity.weight.data, [self.hidden_size, self.input_size])
+        block_orthogonal(self.state_linearity.weight.data, [self.hidden_size, self.hidden_size])
+
+        self.input_linearity.bias.data.fill_(0.0)
```

Comment:
@OyvindTafjord is making a ["forget bias init"](https://github.com/allenai/allennlp/compare/master...OyvindTafjord:rnn_forget_init) for pytorch RNNs.  We should maybe just call it `block_constant`, or something, and use it here.  Just a note, nothing to change now.